### PR TITLE
feat(PoweredBy): update component logo

### PR DESCRIPTION
**Pull request by FabienMotte**
**Summary**

This PR updates the SVG of the `<PoweredBy>` widget component for `react-instantsearch-dom` and `react-instantsearch-hooks-web` and test snapshots.

### CODE REVIEW

**Comment by sarahdayan** [comment]
I see a missing space between the text and logo on React InstantSearch v6.

Capture d’écran 2022-10-21 à 13 09 25

**Comment by FabienMotte** [comment]
> I see a missing space between the text and logo on React InstantSearch v6 ([sandbox](https://codesandbox.io/s/react-instantsearch-app-forked-e7r6fs)).
> 
> Capture d’écran 2022-10-21 à 13 09 25

It looks like we have a `margin-left: 0.3rem` on RIS Storybook but I'm not sure where that's coming from
image

**Comment by sarahdayan** [comment]
@FabienMotte It's from the Algolia theme: https://github.com/algolia/instantsearch-specs/blob/3044c2b7c99e7637403c8a2ea2ff70ec4d112f95/src/styles/themes/algolia.scss#L464-L466 while the [sandbox](https://codesandbox.io/s/react-instantsearch-app-forked-wmw1yb) uses the Satellite theme.

**Comment by FabienMotte** [comment]
> @FabienMotte It's from the [Algolia theme](https://github.com/algolia/instantsearch-specs/blob/3044c2b7c99e7637403c8a2ea2ff70ec4d112f95/src/styles/themes/algolia.scss#L464-L466) while the [sandbox](https://codesandbox.io/s/react-instantsearch-app-forked-wmw1yb) uses the Satellite theme.

Oh I see, so we had the issue before: https://codesandbox.io/s/react-instantsearch-app-forked-xyf9lu
How do you think we should solve this?

File: packages/react-instantsearch-dom/src/components/PoweredBy.js [hunk 1]
```diff
@@ -7,37 +7,10 @@ import { createClassNames } from '../core/utils';
...
-      x2="434.145"
-      y1="404.15"
-      y2="409.85"
-      gradientUnits="userSpaceOnUse"
-      gradientTransform="matrix(94.045 0 0 -94.072 -40381.527 38479.52)"
-    >
-      <stop offset="0" stopColor="#00AEFF" />
-      <stop offset="1" stopColor="#3369E7" />
-    </linearGradient>
+  <svg viewBox="0 0 512 117" className={cx('logo')}>
```
[the remark this answers is not in the document]
Comment (Haroenv) [reply packages/react-instantsearch-dom/src/components/PoweredBy.js:10]: the same height IMO

**Review by Haroenv — CHANGES_REQUESTED** [review]
leaving a review so I know when height is added again :)

**Comment by sarahdayan** [comment]
@FabienMotte Since the issue was already here, we can fix it separately.

Let's make sure we constrain the size of the SVG in this PR, then feel free to open another one to add the margin (can be done after this one is merged).

File: packages/react-instantsearch-dom/src/components/PoweredBy.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (FabienMotte) [reply packages/react-instantsearch-dom/src/components/PoweredBy.js]: Thanks to both of you!

**Comment by Haroenv** [comment]
looks like the examples test snapshot needs to be updated: https://app.circleci.com/pipelines/github/algolia/react-instantsearch/3384/workflows/73addfe9-b508-4795-9235-d40303bc5934/jobs/58582

**Comment by FabienMotte** [comment]
@Haroenv I did update them but forgot to build before, doing it right now

**Review by Haroenv — APPROVED** [review]

**Pull request merged**